docs/Code.md

```diff
@@ -1,8 +1,9 @@
 ---
+layout: default
 title: "Tallwave Guidelines — Code"
 ---
 
-[&larr; Home](/guidelines)
+[&larr; Home]({% if site.github.is_project_page %}/guidelines{% endif %}/)
```

Comment:
Is it possible to add variables to `site` to make lines like this a little easier?

Like:

``` ruby
{%
  site.root = if site.github.is_project_page then '/guidelines' else ''
%}

```

Then you could have links like `[whatever]({% site.root %}/thing)`.

(my ruby is super rust too)